### ISSUE TITLE
Support for no primary key

### ISSUE THREAD

**Issue by dakorpar** [contributte/datagrid#696]
Sometimes I have really simple view only datagrids that don't have any primary key, currently I'm setting it to any column although it has duplicates works ok, but IMHO this should be supported.

**Comment by rpajik** [comment]
Agree, when I want to show data from View, it's not possible now, because there is not index support on Views. Materialized View should by workaround now, but it's not comfortable.
